code/modules/reagents/reagent_containers/glass_containers.dm

```diff
@@ -215,7 +217,7 @@
 
 /obj/item/reagent_containers/glass/beaker/large
 	name = "large beaker"
-	desc = "A large beaker. Can hold up to 100 units."
+	desc = "A large glass beaker, now twice as big!."
```

Comment:
"now twice as big" makes it seem like its comparing to itself. I think something like this is better.
```suggestion
	desc = "A large glass beaker with twice the capacity of a normal beaker."
```

Reply:
and also having a really long examine text can be a bit awkward

Reply:
Its less than 1 line, theres some examines that are 10-30 lines. This is fine on length. If you want you could just cut the description down to
`desc = "A large glass beaker"`

Reply:
And theres the "!." typo at the end too if you choose not to commit this review.

Reply:
but I will say that this is an item that will be examined often so I don't really want to fill up the chat with examine messages.

Reply:
I think I will stick with the twice as big

Reply:
> but I will say that this is an item that will be examined often so I don't really want to fill up the chat with examine messages.

Trust me, this wont fill up the chat. Just take a look at modsuits or APCs if you want to see a bad example of examines being too long

Reply:
Hmm ok, I'll go with it